frontend/src/components/story/StoryCard.tsx

```diff
@@ -29,9 +40,85 @@ const StoryCard = ({
 
     return originalYoutubeLink.replace("watch?v=", "embed/");
   };
+  const handleError = (errorMessage: string) => {
+    alert(errorMessage);
+  };
+
+  const ASSIGN_REVIEWER = gql`
```

Comment:
nit: move `ASSIGN_REVIEWER, AssignReviewer, CREATE TRANSLATION, CreateTranslation` out of `StoryCard `(just before `StoryCardProps `is good). I'll make it a todo to centralize gql calls in a different file in the future.